### PR TITLE
:memo: docs(self hosting): Add deployment link for Alibaba Cloud

### PR DESCRIPTION
**Pull request by LYH-RAIN**
* 📝 doc : Add self-hosting support for Alibaba Cloud.

### CODE REVIEW

File: README.md
Comment (arvinxx) [inline README.md:1]: not recommend to add it in the English README, due to the deploy link is a Chinese product page, not an international page. Please revert this upadte or use an international version

**Review by arvinxx — CHANGES_REQUESTED** [review]

File: README.md
Comment (LYH-RAIN) [reply README.md:1]: The quick deployment link supports both Alibaba Cloud's domestic and international sites, and overseas users will automatically access the quick deployment link for the international site.

Comment (LYH-RAIN) [reply README.md:1]: image
image
Like this. Additionally, the deployment instructions and tutorials for Alibaba Cloud's international site are provided in English only.

Comment (arvinxx) [reply README.md:1]: > The quick deployment link supports both Alibaba Cloud's domestic and international sites, and overseas users will automatically access the quick deployment link for the international site.

How you detect the international users? I try with English locale in browser and oversea VPN, but it just redirect to the domestic site

image

Comment (LYH-RAIN) [reply README.md:1]: Okay, we did overlook this issue, and we will address it later. We will first retract this update and replace it with another PR.
https://github.com/lobehub/lobe-chat/pull/4331

**Pull request closed without merging**